src/Hit/Git/Status.hs

```diff
@@ -0,0 +1,117 @@
+{- | Data types that describe model of the @git diff@ command to display diffs
+in pretty way.
+-}
+
+module Hit.Git.Status
+       ( showPrettyDiff
+       ) where
+
+import Shellmet (($|))
+
+import Hit.ColorTerminal (blueCode, boldCode, cyanCode, greenCode, magentaCode, redCode, resetCode,
+                          yellowCode)
+
+import qualified Data.Text as T
+
+
+-- | Enum that represents all possible types of file modifications.
+data PatchType
+    = Added
+    | Copied
+    | Deleted
+    | Modified
+    | Renamed
+    | TypeChanged
+    | Unmerged
+    | Unknown
+    | BrokenPairing
```

Comment:
Sometimes it seems like all git types and commands etc/ deserve their own library.

Reply:
Yeah, having nice CLI wrapper around `git` would be really nice...